firebase-perf/src/main/java/com/google/firebase/perf/config/ConfigurationConstants.java

```diff
@@ -661,4 +661,39 @@ protected String getMetadataFlag() {
       return "fragment_sampling_percentage";
     }
   }
+
+  protected static final class ExperimentTTID extends ConfigurationFlag<Boolean> {
+    private static ExperimentTTID instance;
+
+    private ExperimentTTID() {
+      super();
+    }
+
+    protected static synchronized ExperimentTTID getInstance() {
+      if (instance == null) {
+        instance = new ExperimentTTID();
+      }
+      return instance;
+    }
+
+    @Override
+    protected Boolean getDefault() {
+      return false;
+    }
+
+    @Override
+    protected String getRemoteConfigFlag() {
+      return "fpr_experiment_as_ttid";
```

Comment:
Similarly, `as` should be `app_start` here.

Reply:
done